setup.py

```diff
@@ -329,7 +317,17 @@ def test_otio():
         'opentimelineview': 'src/opentimelineview',
     },
 
-    install_requires=INSTALL_REQUIRES,
+    # Disallow 3.9.0 because of https://github.com/python/cpython/pull/22670
+    python_requires='>2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, !=3.6.*, !=3.9.0',  # noqa: E501
```

Comment:
this is a nice way to handle versioning!

Reply:
Something like `~=2.7.0, >3.7, <3.10` would be more intuitive IMO, but unfortunately it's not how it works.
```python
>>> packaging.specifiers.SpecifierSet('~=2.7.0, >3.8, <3.10, !=3.9.0').contains('3.8.1')
False
```
:shrug: